modules/specialized-guides/pages/salt/salt-monitoring.adoc

```diff
@@ -0,0 +1,103 @@
+----
+
+Saline is distributed as a separate container image, [literal]``server-saline``, for use with {productname} Server. 
+All related commands and operations should be executed on the {productname} Server container host system.
+
+In addition to its role as a Prometheus exporter, Saline also provides the [literal]``saline-formula``, which simplifies configuration of Prometheus and Grafana dashboards, enhancing monitoring and visualization capabilities.
+
+[[saline-deployment]]
+== Saline deployment
+
+To deploy Saline on the {productname} Server use [command]``mgradm upgrade podman`` with [option]``--saline-replicas 1``.
+Saline can be deployed during the installation of the {productname} Server using [command]``mgradm install podman`` with [option]``--saline-replicas 1``.
+The only possible values for [option]``--saline-replicas`` are ``1`` or ``0``.
+The default port [option]``8216`` can be changed with specifying [option]``--saline-port NNNN`` (where [literal]``NNNN`` is a desired port number) on calling [command]``mgradm upgrade podman``.
+After finishing the deployment [literal]``uyuni-server-saline@0.service`` should be configured on the {productname} Server host system.
+
+== {salt} master configuration recommendations
+
+It is recommended to add [option]``master_stats: True`` and adjust the value of [option]``master_stats_event_iter: 60`` if needed.
+Then restart [command]``salt-master`` to get the detailed statistics of internal {salt} calls.
+This can be help with tuning large scale deployment installations.
+
+== Saline formula
+
+To configure Prometheus and Grafana dashboards, see xref:specialized-guides:salt/salt-formulas-saline.adoc[Saline Formula].
+
+== Removing Saline
+
+To remove Saline from the {productname} Server use [command]``mgradm upgrade podman`` with [option]``--saline-replicas 0``.
+
+= Examples of resulting Grafana dashboards
+
+== Uyuni Server (with Saline) dashboard
+
+The dashboard represents {salt} events with the different views presented on the panels.
+
+[[salt-events-panel]]
+=== Salt Events
+
+The panel represents all the {salt} events grouped by categories. [literal]``Salt events`` represents the total number of all categories of the events.
+
+image::saline-salt-events.png[Salt Events]
+
+=== Minions Activity
+
+The panel provides the minions activity status for the specified interval of times. It shows the amount of minions which were active in last minute, 5 minutes, 15 minutes, 1 hour or considered offline on this moment of time or even ever seen (could be targeted or either responded or not) or active.
+
+image::saline-salt-minions-activity.png[Minions Activity]
+
+=== Salt Events by Tags and Functions
+
+The panel provides the detailed representation of xref:#salt-events-panel[Salt Events] with the event spread by direction. The events which are send to the minions are presented on the positive side of the axis, while the events received from the minions are presented on the negative side.
```

Comment:
I'm not sure if this reference is correct here: `xref:#salt-events-panel[Salt Events]`